acis_thermal_check/main.py

```diff
@@ -271,9 +259,17 @@ def calc_model_wrapper(self, model_spec, states, tstart, tstop, state0=None):
         """
         if state0 is None:
             start_msid = None
+            dh_heater = None
+            dh_heater_times = None
         else:
             start_msid = state0[self.msid]
-        return self.calc_model(model_spec, states, tstart, tstop, start_msid)
+            htrbfn = os.path.join(self.bsdir, 'dahtbon_history.rdb')
```

Comment:
Probably out of scope, but is this likely to be a problem for any historical/regression weeks?

Reply:
I don't think so, but @richardjedgar asked me to include a week where the detector housing heater was on for a test, which I will do.